test/diagnostics/Localization/fr_debug_diagnostic_name.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %empty-directory(%t)
+// RUN: swift-serialize-diagnostics --input-file-path=%S/Inputs/fr.yaml --output-directory=%t/
+// RUN: swift-serialize-diagnostics --input-file-path=%S/Inputs/en.yaml --output-directory=%t/ 2>&1 | %FileCheck %s
+// RUN: not %target-swift-frontend -debug-diagnostic-names -localization-path %S/Inputs -locale fr -typecheck %s 2>&1 | %FileCheck %s --check-prefix=CHECK_NAMES
+
+// CHECK: These diagnostic IDs are no longer availiable: 'not_available_in_def, not_available_in_def_2, not_available_in_def_3, not_available_in_def_4, not_available_in_def_5'
```

Comment:
Could you please remove these diagnostics from yaml in a separate PR?

Reply:
Of course! But can you explain why?

Reply:
I mean shouldn't we be testing/checking on `yaml.unknownIDs`?

Reply:
Since `.def` file is still a source of truth and if it doesn't have it the that diagnostic is no longer viable.

Reply:
Okay... makes sense thank you!